client/client-multi/client-c/build.sh

```diff
@@ -168,7 +108,10 @@ case "$cmd" in
         UNITTESTS_COMPILE=1
```

Comment:
This script is only used for development. I don't see a reason why I should not compile tests.